### PR TITLE
[new release] stringCodepointSplitter (0.0.1)

### PR DESCRIPTION
**Pull request by Yoxem**
Split a string to a list of strings of a character by the unicode codepoint

- Project page: <a href="https://github.com/yoxem/stringCodepointSplitter">https://github.com/yoxem/stringCodepointSplitter</a>

##### CHANGES:

 - add `split_string_by_unicode_codepoint` initially

### CODE REVIEW

File: packages/stringCodepointSplitter/stringCodepointSplitter.0.0.1/opam [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+Split a string to a list of strings of a character by the unicode codepoint.
+It requires module Uutf."""
+maintainer: ["Tan Kian-ting <yoxem@kianting.info>"]
+authors: ["Tan Kian-ting <yoxem@kianting.info>"]
+license: "MIT"
+tags: ["string" "utf8"]
+homepage: "https://github.com/yoxem/stringCodepointSplitter"
+bug-reports: "https://github.com/Yoxem/stringCodepointSplitter/issues"
+depends: [
+  "ocaml"
```
Comment (raphael-proust) [inline packages/stringCodepointSplitter/stringCodepointSplitter.0.0.1/opam:14]:
```suggestion
  "ocaml" {>= "4.07"}
```

Fails with `failed: Unbound module Stdlib` for lower versions.

**Review by raphael-proust — COMMENTED** [review]
I've had a look at the code in your repository and I'm wondering:

Why dynlink uutf? Wouldn't it work to just have dune link against it?

**Comment by Yoxem** [comment]
> I've had a look at the code in your repository and I'm wondering:
> 
> Why dynlink uutf? Wouldn't it work to just have dune link against it?
Iirc
Because the static link didn't linked well and can't be found by the toolchain.

**Comment by raphael-proust** [comment]
I made https://github.com/Yoxem/stringCodepointSplitter/pull/1 which works (there are tests which call a small binary built atop your library) without findlib's dynlink

**Comment by raphael-proust** [comment]
Are you planning to release a v0.0.2 with the newer code in your repo? Should we still merge this PR? Let us know!

**Comment by Yoxem** [comment]
> Are you planning to release a v0.0.2 with the newer code in your repo? Should we still merge this PR? Let us know!

I have create v0.0.2, and make a PR for later version: https://github.com/ocaml/opam-repository/pull/24340

for fear that it will be a redundant PR, I close it.

**Pull request closed without merging**